### DISCUSSION TITLE
runners are Terminating gha-runner-scale-set 0.9.2

### DISCUSSION THREAD

**Discussion by mramachandruni** [actions/actions-runner-controller#3656] [FAQs] [answered]
Hi Team, 
the gha-runner-scale-set runners are terminating as soon as created the pods, 
here is the pod logs

RUNNER 2024-07-12 03:23:39Z INFO JobDispatcher] Set runner/worker IPC timeout to 30 seconds.
[RUNNER 2024-07-12 03:23:39Z ERR  GitHubActionsService] GET request to https://pipelinesghubeus9.actions.githubusercontent.com/rz8qvqxrovPEde5QepqVsamOZo8yxQ9YvzCxhW0cG1t6cOsxVY/_apis/distributedtask/pools/1/messages?sessionId=3f1c687d-8d33-4fe1-be70-27917955ec73&status=Online&runnerVersion=2.316.1&os=Linux&architecture=X64&disableUpdate=true failed. HTTP Status: Forbidden
[RUNNER 2024-07-12 03:23:39Z INFO JobDispatcher] Shutting down JobDispatcher. Make sure all WorkerDispatcher has finished.
[RUNNER 2024-07-12 03:23:39Z INFO Runner] Deleting Runner Session...
An error occured: Runner version v2.316.1 is deprecated and cannot receive messages.
[RUNNER 2024-07-12 03:23:40Z ERR  Terminal] WRITE ERROR: An error occured: Runner version v2.316.1 is deprecated and cannot receive messages.
[RUNNER 2024-07-12 03:23:40Z ERR  Listener] GitHub.DistributedTask.WebApi.AccessDeniedException: Runner version v2.316.1 is deprecated and cannot receive messages.
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Services.WebApi.VssHttpClientBase.HandleResponseAsync(HttpResponseMessage response, CancellationToken cancellationToken)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Services.WebApi.VssHttpClientBase.SendAsync(HttpRequestMessage message, HttpCompletionOption completionOption, Object userState, CancellationToken cancellationToken)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Services.WebApi.VssHttpClientBase.SendAsync[T](HttpRequestMessage message, Object userState, CancellationToken cancellationToken)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Services.WebApi.VssHttpClientBase.SendAsync[T](HttpMethod method, IEnumerable`1 additionalHeaders, Guid locationId, Object routeValues, ApiResourceVersion version, HttpContent content, IEnumerable`1 queryParameters, Object userState, CancellationToken cancellationToken)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.MessageListener.GetNextMessageAsync(CancellationToken token)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.Runner.RunAsync(RunnerSettings settings, Boolean runOnce)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.Runner.RunAsync(RunnerSettings settings, Boolean runOnce)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.Runner.RunAsync(RunnerSettings settings, Boolean runOnce)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.Runner.ExecuteCommand(CommandSettings command)
[RUNNER 2024-07-12 03:23:40Z ERR  Listener]    at GitHub.Runner.Listener.Program.MainAsync(IHostContext context, String[] args)
Runner listener exit with terminated error, stop the service, no retry needed.
Exiting runner...

the runner were running fine till yesterday and there is not changes are made it ,would you please help us what would be the  reason to terminated the runners pod

**Comment by Nastaliss** [comment]
I had this error yesterday, you need to update your docker image to use `2.317.0`.

**Reply by mramachandruni** [reply]
Thanks it is working after upgraded to 2.317.0

**Reply by Nastaliss** [reply, marked as answer]
Perfect :+1:  Can you mark the discussion as answered ?
